app/models/user.rb

```diff
@@ -72,6 +71,7 @@
 #  created_at                  :datetime         not null
 #  updated_at                  :datetime         not null
 #  facebook_id                 :string(255)      indexed
+#  one_signal_id               :string           indexed
```

Comment:
We're going to have multiple clients and it seems like each client which registers will get issued a separate ID.  Could this be moved into a separate table like this:

```
OneSignalPlayer
  User user_id
  String one_signal_id
  Enum<mobile,web> platform
```

That way we can register as many OneSignal IDs as necessary